eng/Directory.Build.Data.targets

```diff
@@ -63,6 +63,11 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" />
   </ItemGroup>
+  
+  <!-- Enable ApiCompat  -->
+  <ItemGroup>
+    <ProjectReference Include="$(RepoRoot)/eng/ApiCompat/ApiCompat.csproj" ReferenceOutputAssembly="false" BuildReference="false" Condition="'$(RunApiCompat)' == 'true'" />
```

Comment:
We should keep a close eye on this to ensure we don't end up restoring here multiple times and that our calls here and in the RunApiCompat target don't end up causing duplicate work.